no/PrivateUnsecuredLoan/example/PrivateUnsecuredLoanApplicationCreated.json

```diff
@@ -1,85 +1,113 @@
+    "eventTime": "2020-05-13T10:29:21+02:00",
+    "extensions": {
+        "comExampleExtension": "value"
     },
-    "contentType" : "application/json",
+    "contentType": "application/json",
     "data": {
-		"application": {
-			"applicant": {
-				"ssn": "11111111111",
+        "application": {
+            "applicant": {
+                "ssn": "11111111111",
                 "customerId": "abc146",
-				"phone": "+4740123456",
-				"secondaryPhone": [],
-				"emailAddress": "example@example.no",
-				"employmentStatus": "PRIVATE_SECTOR",
+                "phone": "+4740123456",
+                "secondaryPhone": [],
+                "emailAddress": "example@example.no",
+                "employmentStatus": "PRIVATE_SECTOR",
                 "employmentStatusSinceYear": 2010,
                 "employmentStatusSinceMonth": 12,
                 "employerName": "Example AB",
                 "employerPhone": "+4740123456",
                 "dependentChildren": 0,
                 "paymentRemark": false,
-				"housingType": "LODGER",
+                "housingType": "LODGER",
                 "housingSinceYear": 2010,
                 "housingSinceMonth": 12,
-				"housingCostPerMonth": 3000,
-				"netMonthlyIncome": 10000,
+                "housingCostPerMonth": 3000,
+                "netMonthlyIncome": 10000,
                 "grossYearlyIncome": 160000,
                 "partnerYearlyIncome": 0,
-				"maritalStatus": "COHABITING",
-				"bankAccount": "12345678901",
-				"citizenships": [
-					"NO",
+                "maritalStatus": "COHABITING",
+                "bankAccount": "12345678901",
+                "citizenships": [
+                    "NO",
                     "SE"
-				],
-				"livedInCountrySinceYear": 1980,
-                "countriesOfResidence": ["NO"],
-				"taxResidentOf": ["SE"],
+                ],
+                "livedInCountrySinceYear": 1980,
+                "countriesOfResidence": [
+                    "NO"
+                ],
+                "taxResidentOf": [
+                    "SE"
+                ],
                 "education": "UNIVERSITY_SHORT",
-				"tentativeAddress": {
-					"firstName": "Bjarne",
-					"lastName": "Bjarnesen",
-					"address": "Fjordgatan 1",
-					"postalCode": "1234",
-					"city": "Oslo"
-				}
-			},
-			"existingLoans": [
-				{
-					"loanAmount": 100000,
-					"monthlyPayment": 3000,
-					"refinanceAmount": 100000,
-					"existingLoanType": "UNSECURED_LOAN",
-					"responsibility": "SHARED",
+                "tentativeAddress": {
+                    "firstName": "Bjarne",
+                    "lastName": "Bjarnesen",
+                    "address": "Fjordgatan 1",
+                    "postalCode": "1234",
+                    "city": "Oslo"
+                }
+            },
+            "existingLoans": [
+                {
+                    "loanAmount": 100000,
+                    "nominalInterestRate": null,
```

Comment:
`null` is not allowed by schema for this field. Since the field is optional it can be omitted here.